driver/controller.go

```diff
@@ -590,12 +590,28 @@ func (d *Driver) ListVolumes(ctx context.Context, req *csi.ListVolumesRequest) (
 
 // GetCapacity returns the capacity of the storage pool
 func (d *Driver) GetCapacity(ctx context.Context, req *csi.GetCapacityRequest) (*csi.GetCapacityResponse, error) {
-	// TODO(arslan): check if we can provide this information somehow
+	volumes, _, err := d.storage.ListVolumes(ctx, &godo.ListVolumeParams{})
+	if err != nil {
+		return nil, err
+	}
+
+	totalCapacity := int64(0)
+	for _, volume := range volumes {
+		if volume.Region.Slug == d.region {
+			totalCapacity += int64(volume.SizeGigaBytes) * 1024 * 1024 * 1024
+		}
+	}
+
+	// Create a new response object
+	resp := &csi.GetCapacityResponse{
+		AvailableCapacity: totalCapacity,
```

Comment:
AFAIU [the spec for `GetCapacity`](https://github.com/container-storage-interface/spec/blob/b01039c563108173c6743aa1410ec11fde7c24fe/spec.md#getcapacity), the availability capacity is not how much storage capacity is currently in use but how much capacity is available to the user.

I believe this is essentially the volume limit (which we still reference to some extent in the `checkLimit` method that is otherwise unused and was missed for removal in #481) minus the currently used capacity.

Reply:
Yes you are correct, We can calculate the used capactity by adding volumesize of volumes whose dropletid is not nil